ambuda/templates/dictionaries/index.html

```diff
@@ -59,5 +59,30 @@ <h1 class="text-xl font-bold my-4">{{ _('Dictionary lookup') }}</h1>
   {% include 'include/dictionary-help.html' %}
 {% endif %}
 </div>
+{% endmacro %}
 
+
+{% block main %}
+<article class="p-4 md:p-0 max-w-4xl mx-auto md:px-2 md:my-24 md:mb-36">
+
+<h1 class="text-xl font-bold my-4">{{ _('Dictionary lookup') }}</h1>
+
+<div x-data="dictionary" class="lg:flex">
+  <div class="flex-1 max-w-[700px]">
+    {{ dictionary_form() }}
+    {{ dictionary_response(query, entries) }}
+  </div>
+
+  <div class="text-sm
+      bg-slate-100 rounded p-2 overflow-hidden
+      mt-4 lg:mt-0 lg:ml-4 lg:w-48">
+    <h3 class="font-bold text-slate-500 mb-2">{{ _('History') }}</h3>
```

Comment:
Hmm ... stepping one level up, I'm ambivalent about this feature's value in general. Copying from Discord:

> This feature is "cool" but after thinking about it, I'm not convinced it's solving a real problem. If [someone] feels strongly that this is a worthwhile feature I'll keep it, otherwise I'll shelve it for now.

While the feature itself is strictly a benefit, it takes up substantial screen real estate. So I'm considering rendering this underneath the `select` options in the style of sanskritdictionary.com.

What do you think?